src/ml/neural_net/style_transfer/mps_style_transfer_pre_processing.h

```diff
@@ -0,0 +1,30 @@
+/* Copyright © 2018 Apple Inc. All rights reserved.
+ *
+ * Use of this source code is governed by a BSD-3-clause license that can
+ * be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+ */
+
+#import <Accelerate/Accelerate.h>
+#import <Foundation/Foundation.h>
+#import <Metal/Metal.h>
+#import <MetalPerformanceShaders/MetalPerformanceShaders.h> 
+
+#import <ml/neural_net/mps_descriptor_utils.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+API_AVAILABLE(macos(10.14))
+@interface TCMPSStyleTransferPreProcessing : NSObject 
+
+@property (nonatomic) MPSNNImageNode *output;
+
+- (instancetype) initWithParameters:(NSString *)name
+                          inputNode:(MPSNNImageNode *)inputNode
+                         scaleNode:(MPSNNImageNode *)scaleNode
```

Comment:
Mega-nit: Align parentheses? Maybe just use git-clang-format? (Shreya has started doing this in some of her PRs...)

Reply:
Perhaps, I refrain from using `clang-format` on Objective-c code. It just messes things up sometimes. I aligned the parentheses in all of the files I found mis-aligned!